src/modules/redesign/i18n/fr_FR/verband/protokoll_types.ts

```diff
@@ -0,0 +1,20 @@
+export default {
+    mission: { regex: /Large mission started/ },
+    event: { regex: /Alliance event started/ },
+    appl_accepted: { regex: /Added to the alliance/ },
+    appl_declined: { regex: /Application denied/ },
+    deny_appl: { regex: /Not allowed to apply for the alliance/ },
+    allow_appl: { regex: /Allowed to apply for the alliance/ },
+    left: { regex: /Left the alliance/ },
+    kicked: { regex: /Kicked from the alliance/ },
+    set_chatban: { regex: /Chat ban set/ },
+    remove_chatban: { regex: /Chat ban removed/ },
+    added_role: { regex: /Set as .*?/, title: 'Role added' },
+    removed_role: { regex: /Removed as .*?/, title: 'Role removed' },
+    start_schooling: { regex: /Created a course/ },
+    complete_schooling: { regex: /Course completed/ },
+    start_extension: { regex: /Expansion started/ },
+    complete_extension: { regex: /Expansion finished/ },
+    build: { regex: /Building constructed/ },
+    demolish: { regex: /Building destroyed/ },
+};
```

Comment:
regexes here are taken from the alliance log files. You may ask an admin of your alliance if they can share some of the texts. Don't worry if you don't find all texts, we can add them later.